transcom/mymove
# MAIN B-21376

[INT PR](https://github.com/transcom/mymove/pull/14102)
[INT PR 2](https://github.com/transcom/mymove/pull/14263)
[INT PR 3](https://github.com/transcom/mymove/pull/14276)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21376)

> [!IMPORTANT]
> This PR contains the `main` branches of [B-21374](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21374) & [B-21375](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21375)
> You can review this PR, but do not merge until those have been merged

## Summary

When service members are creating unaccompanied baggage shipments, we need to let them know what their allowance is as they create the shipment. This is as simple as updating the helper text that appears at the top of the page letting them know what their allowance is. 

### How to test

1. In your `.envrc` you'll need to turn `FEATURE_FLAG_UNACCOMPANIED_BAGGAGE` to `true`
2. Think of a duty location and go turn that `address.is_oconus` to `true` 
3. Fire up your `client` and `server`
4. Log in as a customer
5. Create your profile and go to add some orders
6. That duty location that you set up to be OCONUS, make that your origin/destination - you really just need the one
7. Finish setting up your orders
8. Go to your database and find the `entitlements` table
9. Find the row that was just created and update the `ub_allowance` column to be whatever integer you want
10. Go and set up a shipment
11. Select the UB shipment option
12. You should now see the helper text letting you know what your allowance is at the top of the page
13. Delete the value you just put in the `ub_allowance` column in the database
14. Refresh the screen - you should now see the default helper text that's a little more vague but still relevant to the UB shipment

## Screenshots

![Screenshot 2024-11-04 at 9 14 01 AM](https://github.com/user-attachments/assets/50330910-5054-44c5-a825-337f27b2fde4)

![Screenshot 2024-11-04 at 9 30 17 AM](https://github.com/user-attachments/assets/84cbc664-d821-4416-9318-e532d9c03aec)

![Screenshot 2024-11-04 at 9 14 22 AM](https://github.com/user-attachments/assets/a1f9dec1-e2cc-49a4-8ce9-d89ef0add55e)


## [Issue ticket](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A1034466)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21376)

## Summary

Wanted a ToolTip. Here's yer ToolTip.

### How to test

1. Access MM as customer (need to set up `.envrc` to allow for UB shipments
2. Fill in orders info to qualify you for a UB shipment - submit
3. Go to your home page, you should now see the UB allowance & a tooltip at the top next to the weight allowance

## Screenshots
![Screenshot 2024-11-25 at 9 50 09 AM](https://github.com/user-attachments/assets/300f6059-269c-4bcb-88d0-db4e8aa4991e)
![Screenshot 2024-11-25 at 9 50 47 AM](https://github.com/user-attachments/assets/e9ec8a14-f014-44af-9a43-dac6b3fd7b1e)
